packages/js-core/src/app/lib/attributes.ts

```diff
@@ -19,6 +19,13 @@ export const updateAttribute = async (key: string, value: string): Promise<Resul
   const res = await api.client.attribute.update({ userId, attributes: { [key]: value } });
 
   if (!res.ok) {
+    // @ts-expect-error
+    if (res.error.status === 403) {
+      logger.error(
+        "This attribute could not be set due to the maximum number of attribute classes being reached for this environment."
```

Comment:
Can we take a more generic approach here to set this information in the endpoint and print the message in the js-core package instead of assuming that every 403 will be because of this reason?